.gitignore

```diff
@@ -1,20 +1,8 @@
-# We filter only autotools artefacts.
 # Use a separate build directory!
+# Have a look at https://wiki.gnucash.org/wiki/Documentation_Improvement#Initial_Steps
 # For other purposes adjust "$XDG_CONFIG_HOME/git/ignore!
 
 #absolute:
-/INSTALL
-/aclocal.m4
-/autom4te.cache/
-/config.guess
-/config.sub
-/configure
-/install-sh
-/ltmain.sh
-/missing
-/build
-/guide/pt/Auxiliary files/
-/home/pmra/Documents/6.Translations/GnuCash/gnucash-docs/gnucash-docs/guide/pt/figures100
 
 #relative:
 Makefile.in
```

Comment:
This can go, too—another autotools artefact.

As the file has no entries then, we can also remove the absolute/relative header lines.